File: keybind-common/src/main/java/io/github/solusmods/eternalcore/keybind/EternalCoreKeybindClient.java
package io.github.solusmods.eternalcore.keybind;

import dev.architectury.event.events.client.ClientLifecycleEvent;
import io.github.solusmods.eternalcore.keybind.api.KeybindingManager;

public class EternalCoreKeybindClient {
    public static void init() {
        ClientLifecycleEvent.CLIENT_SETUP.register(instance -> KeybindingManager.init());
    }
}

File: keybind-common/src/main/kotlin/io/github/solusmods/eternalcore/keybind/EternalCoreKeybindClient.kt
package io.github.solusmods.eternalcore.keybind

import dev.architectury.event.events.client.ClientLifecycleEvent
import io.github.solusmods.eternalcore.keybind.api.KeybindingManager
import net.minecraft.client.Minecraft

object EternalCoreKeybindClient {
    fun init() {
        ClientLifecycleEvent.CLIENT_SETUP.register(ClientLifecycleEvent.ClientState { instance: Minecraft -> KeybindingManager.init() })
    }
}
